db/db.go

```diff
@@ -0,0 +1,56 @@
+package db
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/dialect/sqlitedialect"
+	"github.com/uptrace/bun/extra/bundebug"
+)
+
+var DB *RootDB
+
+type RootDB = bun.DB
+
+func NewDB(dbPath string) (RootDB, error) {
+	sqldb, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return RootDB{}, err
+	}
+
+	// create the database if it doesn't exist
+	// XXX this'll be in bun migrations.
```

Comment:
It sounds like we're fine using bun migrations IRL, but I have zero intention of allowing migrations on this webserver.

Blowing away the sqlite.db and starting from scratch results in degraded response times for like 30 min, and I expect that to happen maybe once a year.  Not worth the complexity.

Reply:
yep I added this comment mostly for illustration db.go won't have to contain a description of all models in the database.